sources/HUBComponentImageLoadingContext.h

```diff
@@ -51,7 +53,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithImageType:(HUBComponentImageType)imageType
                   imageIdentifier:(nullable NSString *)imageIdentifier
                 wrapperIdentifier:(NSUUID *)wrapperIdentifier
-                       childIndex:(nullable NSNumber *)childIndex HUB_DESIGNATED_INITIALIZER;
+                       childIndex:(nullable NSNumber *)childIndex
+                        timestamp:(CFTimeInterval)timestamp HUB_DESIGNATED_INITIALIZER;
```

Comment:
How about changing the type of this parameter and the associated property to `NSTimeInterval`? That would keep the API cleaner, abstracting away that we currently use Core Animation function. Should make things nicer to use from Swift.